Updates event source for four problem events

lamagnifica opened the pull request:
## [DOC-3229](https://openedx.atlassian.net/browse/DOC-3229)

Per research by @muhammad-ammar and team, changing source of four problem-related events from server to browser.
https://openedx.atlassian.net/wiki/display/AN/CAPA+events+after+a11y+changes

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [ ] Subject matter expert: @stroilova 
- [ ] Subject matter expert: @muzaffaryousaf 
- [x] Subject matter expert: @muhammad-ammar 
- [ ] Subject matter expert: @brianhw 
- [x] Doc team review (copy edit): @catong @pdesjardins 
- [ ] Product review: @sstack22 
- [ ] Partner support: 
- [ ] PM review: 

FYI: 

### Testing

- [ ] Ran ./run_tests.sh without warnings or errors

### Post-review

- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

catong commented on en_us/data/source/internal_data_formats/tracking_logs.rst:
```diff
@@ -3200,9 +3184,9 @@ successfully.
 .. no sample to check
 
-The server emits ``showanswer`` events when the answer to a problem is shown.
+The browser emits ``showanswer`` events when the answer to a problem is shown.
```
Any way of changing this from passive to active? Are there specific triggers to answers being shown?

lamagnifica replied:
There are, but the user triggers a series of activities, and aren't the immediate cause of the event. So if a learner select "Show Answer" the browser does a few steps, talks to the server, the server responds, and THEN emits an event/shows the answer to the learner. In this file, active voice is extremely hard to use accurately. If you think it's very important, I'll open a separate story to tackle it for the entire file, but it's likely to keep getting kicked down the street...

catong replied:
Ah, thanks for the explanation. I'm fine with existing wording if it's deliberate. Does this mean "Reset" above is different, and the event is emitted by the user selecting "Reset"?

lamagnifica replied:
Yeah. You could be shown an answer because you selected Show Answer, or you could be shown the answer after the final attempt is made, or ... all the settings and otpions, all the capa problems, oh my

muhammad-ammar replied:
@lamagnifica `showanswer` event is emitted from server side.

lamagnifica replied:
thank you @muhammad-ammar ! I completely missed that on the wiki page.

catong commented:
@lamagnifica LGTM with one question on "showanswer".

lamagnifica commented:
Updated showanswer with the correct info from the wiki page (thanks @muhammad-ammar ). I also realized that the example problem at the beginning of the file was an old problem_check, so I replaced it with the newly generated example.

muhammad-ammar commented:
@lamagnifica LGTM 👍
